src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_INDEX = new Random().nextInt(5);
+    private static final int FIRST_SIDE = new Random().nextInt(1,10);
+    private static final int SECOND_SIDE = new Random().nextInt(1,10);
```

Comment:
Remove these constants. You generate random sides once, and they stay the same for all the figures produced via this class. Instead, create a private helper method for generating geometric properties. Make 10 a constant. Make Random a field (not constant).